applications/geoclaw/bowl_slosh/regressions.sh

```diff
@@ -3,7 +3,8 @@
 application=$FCLAW_APPLICATIONS_BUILD_DIR/geoclaw/bowl_slosh/bowl_slosh
 
 # change to source dir for working directory
-cd $FCLAW_APPLICATIONS_SRC_DIR/geoclaw/bowl_slosh/regression
+cd $FCLAW_APPLICATIONS_SRC_DIR/geoclaw/bowl_slosh/
 
 # run programs, exit script with nonzero on failure (or else script will exit with value of last program run)
-$FCLAW_MPIRUN $FCLAW_MPI_TEST_FLAGS $application -F regression.ini || exit 1
+$FCLAW_MPIRUN $FCLAW_MPI_TEST_FLAGS $application -F regression.ini --user:claw-version=4 || exit 1
```

Comment:
This was discussed in a previous PR : The Geoclaw code does not need versions 4 and 5; it runs version 5 by default.